my-app/src/createFolder.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./css/createFolder.css"
+import ReactMultiSelectCheckboxes from 'react-multiselect-checkboxes'
+
+class CreateFolder extends React.Component{
+
+    render(){
+        const options = [
+            { label: 'File 1', value: 1},
+            { label: 'File 2', value: 2},
+            { label: 'File 3', value: 3},
+            { label: 'File 4', value: 4},
+            { label: 'File 5', value: 5},
+            { label: 'File 6', value: 6},
+
+        ];
+        return(
+            <div class = "row">
+                <h1 id = "createFolderHeader">Create Your own folder</h1>
```

Comment:
The component layout on this page is weird. Can you make it look more like the "Create Your own File" page?